enclave-manager/web/src/components/RelativeDateTime.tsx

```diff
@@ -0,0 +1,35 @@
+import { DateTime } from "luxon";
+import { Text, TextProps, Tooltip } from "@chakra-ui/react";
+import { isDefined } from "../utils";
+import { useEffect, useState } from "react";
+
+type RelativeDateTimeProps = TextProps & {
+  dateTime: DateTime | null;
+};
+
+export const RelativeDateTime = ({ dateTime, ...textProps }: RelativeDateTimeProps) => {
```

Comment:
maybe consider using react-moment. I think it does the same?

https://www.npmjs.com/package/react-moment

Reply:
interesting library - I've not come across it before. The main thing I like about luxon over moment is immutable objects.

Reply:
That's neat. I think you can still use luxon here, it's just the formatting of the timestamp that you get for free with react-moment. I.e. you can skip the manual handling of state and timeouts etc.